pkg/registration/spoke/spokeagent.go

```diff
@@ -517,6 +510,18 @@ func (o *SpokeAgentConfig) RunSpokeAgentWithSpokeInformers(ctx context.Context,
 		)
 	}
 
+	if o.registrationOption.RegistrationAuth == CsrAuthType {
```

Comment:
```suggestion
	if o.registrationOption.RegistrationAuth != AwsIrsaAuthType {
```
This might have less chance of regression.

Reply:
Resolving. See Qiu Jian's comment: https://github.com/open-cluster-management-io/ocm/pull/736#issuecomment-2517827947 Thanks.